_includes/fragments/header.html

```diff
@@ -14,7 +14,7 @@
                     </div>
                     <div class="collapse navbar-collapse" id="navbar-collapse">
                         <ul class="nav navbar-nav">
-                            {% if page.url != '/index.html' %}
+                            {% if page.url != '/' %}
```

Comment:
I tested locally with URLs `http://localhost:4000/` and `http://localhost:4000/index.html` and both resolve to a `page.url` value of `/`.